src/dashboard/src/components/helpers.py

```diff
@@ -204,6 +207,35 @@ def get_server_config_value(field):
     except:
         return ''
 
+def get_atom_levels_of_description(clear=True):
+    url = get_setting('dip_upload_atom_url')
+    if not url:
+        raise Exception("AtoM URL not defined!")
```

Comment:
Do we want an `AtomException`? I guess there's no harm in adding a very specific exception.